### PR TITLE
feat: use critical-hds-styles from shared assets

### PR DESCRIPTION
**Pull request by nikomakela**
LIIKUNTA-612 (HH-311 HH-335).

Use the Critical HDS Styles from the shared assets,
so it is publicly available for the `_document.tsx` to use.
The `_document.tsx` now uses a <link>-tag to import
the CSS styles to the application.

When the CSS is imported from the shared-assets,
it's also available for every application (from that 1 generated file).

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr632.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr632.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr632.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr632.dev.hel.ninja** :rocket::rocket::rocket:

File: README.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ To enable the support for symbolic links on Windows 10 you need to:
...
 
-By executing `yarn g:hds-critical-styles-create`, the `critical-hds-styles.css` file will be created in every app and package where needed. The command uses the yarn workspaces and the related apps' `yarn hds:critical-styles:create` -command to do that.
+By executing `yarn g:hds-critical-styles:create`, the `critical-hds-styles.css` file will be created. The root package.json uses the static assset styles path ([./static/assets/styles/critical-hds-styles.css](./static/assets/styles/critical-hds-styles.css)) to offer the same file for every application. The [critical-hds-styles.css](./static/assets/styles/critical-hds-styles.css) is then used by the app's `_document.tsx` which creates a `<link rel="stylesheet">`-tag out of it.
```
Comment (karisal-anders) [inline README.md:263]: typo `assset` -> `asset`

**Review by karisal-anders — APPROVED** [review]

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr632.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr632.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr632.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr632.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**